.eslintrc.js

```diff
@@ -0,0 +1,98 @@
+module.exports = {
+  'env': {
+    'browser': true,
+    'es6': true,
+  },
+  'extends': 'standard-with-typescript',
+  'ignorePatterns': [
+    '.eslintrc.js'
+  ],
+  'plugins': [
+    'licenses'
+    ],
+  'overrides': [
+  ],
+  'parserOptions': {
+    'project': './tsconfig-lint.json'
+  },
+  'rules': {
+    'licenses/header': [
+      2,
+      {
+        'tryUseCreatedYear': true,
+        'comment': {
+          'allow': 'both',
+          'prefer': 'line'
+        },
+        'header': [
+          'Copyright (c) {YEAR} The Brave Authors. All rights reserved.',
+          'This Source Code Form is subject to the terms of the Mozilla Public',
+          'License, v. 2.0. If a copy of the MPL was not distributed with this file,',
+          'You can obtain one at https://mozilla.org/MPL/2.0/.'
+        ],
+        'altHeaders': [
+          [
+            'Copyright (c) {YEAR} The Brave Authors. All rights reserved.',
+            '* This Source Code Form is subject to the terms of the Mozilla Public',
+            '* License, v. 2.0. If a copy of the MPL was not distributed with this file,',
+            '* You can obtain one at https://mozilla.org/MPL/2.0/.'
+          ],
+          [
+            'This Source Code Form is subject to the terms of the Mozilla Public',
```

Comment:
You can remove this one, if all your headers use the proper format. It's only for legacy purposes in Brave-Core

Reply:
Cool! Done.